src/components/measurements/index.tsx

```diff
@@ -182,14 +205,20 @@ const MeasurementsPlot = ({height, width, showLegend, setPanelTitle}) => {
     groupedMeasurements
   });
   // Cache handleHover function to avoid extra useEffect calls
-  const handleHover = useCallback((data, dataType, mouseX, mouseY, colorByAttr=null) => {
+  const handleHover = useCallback((
+    data: Measurement | { mean: number, standardDeviation: number },
+    dataType: "measurement" | "mean",
+    mouseX: number,
+    mouseY: number,
+    colorByAttr: string = null
+  ): void => {
     let newHoverData = null;
     if (data !== null) {
       // Set color-by attribute as title if provided
       const hoverTitle = colorByAttr !== null ? `Color by ${getColorByTitle(colorings, colorBy)} : ${colorByAttr}` : null;
       // Create a Map of data to save order of fields
       const newData = new Map();
-      if (dataType === "measurement") {
+      if (dataType === "measurement" && measurementIdSymbol in data) {
```

Comment:
I couldn't understand d320d259870e174691b7ae884e5710d38889fbbc. Do you think something like this makes sense instead? da3c4128b207ca1f159442c62fd6aa8b95ebea7f

Reply:
Thanks for pointing out this is unclear! Would it be more readable if I just remove the `dataType` param and just check the type of the `data`. 

```javascript
if (isMeasurement(data)) {
  ...
} else if (isMeanAndStandardDeviation(data)) {
  ...
} else {
  throw ...
}

Reply:
Updated in https://github.com/nextstrain/auspice/pull/1910/commits/435b3f86dad7ea02b4d471d827ecb48452b60256